### ISSUE TITLE
Allow commenting out multiple lines

### ISSUE THREAD

**Issue by xorinzor** [savonet/liquidsoap#2776]
**Is your feature request related to a problem? Please describe.**
Currently it seems you can only comment out single lines using `#`.
It would be very nice if we could also use `/* comments */` to comment out multiple lines.

**Describe the solution you'd like**
A way to easily comment out multiple lines without having to edit every line

**Describe alternatives you've considered**
N/A

**Comment by toots** [comment]
This is a very legit feature that I also would like to have at some point.

**Comment by smimram** [comment]
My only concern is about style: usually `/* ... */` goes with `//...` not with `# ...`. Does python have a multiline comment for instance?

**Comment by vitoyucepi** [comment]
Nope, python doesn't have block comments at all.
Though there are multiline docstrings.

**Comment by toots** [comment]
Yeah docstrings can be used a de-factor multiline comments in python when they are not immediately preceding a method:

```python
# Write Python code here
 
""" Multi-line comment used
print("Python Comments") """
 
print("Mathematics")
```
Do we want the same syntax is another question.. 😅

We could go the same way as python for adoption & simplicity but, personally, triple quotes feels like a syntax for multiline strings not comments.

In shell, another "language" with `#` for comments, multiline are denoted by `: '`:
```shell
: '
echo"This doesn't echo"

echo"Even this doesn't"
 '
echo "OK, this is working with : '"
```

Clearly not for us.

In ruby and perl:
```ruby
=begin 
This is slightly long
multi line comment in Ruby
=end
```
Again, not a fan.

Since there is no consensus nor seemingly naturally good syntax for those, I suppose we could introduce ours..

How about:
```
# This is a single line comment

#--
This is a
multiline 
comment
#--
```
This is bit reminiscent of `<--` in HTML, relatively fast to type and pretty clear that this marks a delimitation

**Comment by vitoyucepi** [comment]
Another from *shell* using *heredoc*
```sh
<<comment
echo"This doesn't echo"

echo"Even this doesn't"
comment
```

*Jinja2*, not a programming language, but a template engine
```j2
{# note: commented-out template because we no longer use this
    {% for user in users %}
        ...
    {% endfor %}
#}
```

**Comment by vitoyucepi** [comment]
Implemented in #2792.

This feature is available in *main* and in *rolling-release-v2.2.x* branches.
[Documentation for the development version.](https://www.liquidsoap.info/doc-dev/language.html#code-comments)
